openai-chat.el

```diff
@@ -92,6 +92,11 @@ STREAM, STOP, MAX-TOKENS, PRESENCE-PENALTY, FREQUENCY-PENALTY, and LOGIT-BIAS."
   :type 'number
   :group 'openai)
 
+(defun openai--resolve-key (key)
+  (if (functionp key)
+      (funcall key)
+    key))
```

Comment:
We are sharing this function across all files, you might want to move this to `openai.el`. And make sure to add the document string for the function! :D